### DISCUSSION TITLE
Proxmox VE Post Install

### DISCUSSION THREAD

**Discussion by circa1741** [tteck/Proxmox#186] [Ideas]
Some suggested additions to the Post Install script:

**Prevent auto-sleep**
systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
systemctl restart systemd-logind.service

**Configure laptop lid status** (useful for laptops)
nano /etc/systemd/logind.conf
HandleLidSwitch=ignore
update-grub

**Configure swappiness**
nano /etc/sysctl.conf
vm.swappiness = 0
sysctl -p

Script v3 will be nice to give the users options. Thanks.

**Comment by nelssimo** [comment]
Was wondering if this has already been implemented in any way already

**Reply by tteck** [reply]
No
